timelock-impl/src/test/java/com/palantir/atlasdb/timelock/TimeLockResourceTest.java

```diff
@@ -113,4 +156,24 @@ private String uniqueClient() {
         return UUID.randomUUID().toString();
     }
 
+    private void assertNumberOfActiveClientsIs(int expected) {
+        assertThat(getGaugeValueForTimeLockResource(TimeLockResource.ACTIVE_CLIENTS))
+                .isEqualTo(expected);
+    }
+
+    private void assertMaxClientsIs(int expected) {
+        assertThat(getGaugeValueForTimeLockResource(TimeLockResource.MAX_CLIENTS))
+                .isEqualTo(expected);
+    }
+
+    private int getGaugeValueForTimeLockResource(String gaugeName) {
+        Object value = Optional.ofNullable(metricsManager.getRegistry()
+                .getGauges()
+                .get(new StringJoiner(".").add(TimeLockResource.class.getCanonicalName())
```

Comment:
Surely String concatenation is cleaner than StringJoiner in this instance?

Reply:
Yeah, that's true 🤦‍♂️ 
I think when I wrote the first part I was thinking I might need to string a bunch of pieces together, but `getCanonicalName()` already has most of the things in order for you.